.ebextensions/nginx.config

```diff
@@ -332,31 +332,6 @@ files:
 
         charset utf-8;
 
-        # With Content Security Policy (CSP) enabled (and a browser that supports it (http://caniuse.com/#feat=contentsecuritypolicy),
-        # you can tell the browser that it can only download content from the domains you explicitly allow
-        # CSP can be quite difficult to configure, and cause real issues if you get it wrong
-        # There is website that helps you generate a policy here http://cspisawesome.com/
-        add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' https://dap.digitalgov.gov https://www.google-analytics.com; style-src 'self' 'unsafe-inline'; img-src 'self' data: https://www.google-analytics.com *.tile.openstreetmap.org; frame-ancestors 'none'; frame-src https://www.youtube.com;" always;
-
-        # MIME type sniffing security protection
-        #	There are very few edge cases where you wouldn't want this enabled.
-        add_header X-Content-Type-Options nosniff always;
-
-        # The X-Frame-Options header indicates whether a browser should be allowed
-        # to render a page within a frame or iframe.
-        add_header X-Frame-Options SAMEORIGIN always;
```

Comment:
Do we ever iframe this page internally? Might be worth changing to DENY; either way, we should preserve this on static content.